networks/motion/CNN2D/motion_head_CNN2D.py

```diff
@@ -28,15 +28,14 @@
 def createModel(patchSize):
     cnn = Sequential()
     cnn.add(Convolution2D(32,
-                            14, 
-                            14, 
-                            init='he_normal',
+                            (14, 14),
+                            kernel_initializer='he_normal',
                            # activation='sigmoid',
                             weights=None,
-                            border_mode='valid',
-                            subsample=(1, 1),
-                            W_regularizer=l2(1e-6),
-                            input_shape=(1, patchSize[0,0], patchSize[0,1])))
+                            padding='valid',
```

Comment:
changes not necessary
leave it as in master

Reply:
did you change them for compatibility or why?

Reply:
Yes, I changed it as keras recommended for compatibility. Later I'll focus on motion_all_CNN2D and I'll abandon this change.